test/acl.test.js

```diff
@@ -393,20 +393,18 @@ describe('security ACLs', function() {
 });
 
 describe('access check', function() {
-  var app;
-  before(function() {
-    app = loopback();
-    app.set('remoting', { errorHandler: { debug: true, log: false }});
-    app.use(loopback.rest());
-    app.enableAuth();
-    app.dataSource('test', { connector: 'memory' });
```

Comment:
I think we misunderstood each other. Keep the `app` creation in the `before` hook, because that part is typically shared by all test. Keep the line `var MyTestModel = ...` in the test, because the model is specific to the test case.